### PR TITLE
Fixed incorrect docstring in run_glm()

### PR DESCRIPTION
**Pull request by kchawla-pi**
As per https://github.com/nistats/nistats/issues/369#issuecomment-512987725

### CODE REVIEW

**Comment by bthirion** [comment]
LGTM. B

**Comment by kchawla-pi** [comment]
The CI is failing for some reason. Will check it out tomorrow.

**Comment by jeromedockes** [comment]
you can also fix the "Attributes" section of `FirstLevelModel`which has the same problem

**Comment by kchawla-pi** [comment]
I will need a windows machine to diagnose this. Will do it at home.

**Comment by jeromedockes** [comment]
as pointed out in this comment you can also fix the `labels` and `results` attribute names

https://github.com/nistats/nistats/issues/369#issuecomment-513253665

**Comment by jeromedockes** [comment]
> I will need a windows machine to diagnose this. Will do it at home.

not related to the OS, you just need a recent pandas version. `DataFrame.from_csv` has been removed.
luckily it seems to be used only in the test so you can just change (to replace `DataFrame().from_csv` with `read_csv`) this line

https://github.com/nistats/nistats/blob/9fa2e0e7cd8a0deed5f57d9645d080b64e671de2/nistats/tests/test_dmtx.py#L469

File: nistats/first_level_model.py [hunk 1]
```diff
@@ -114,7 +114,7 @@ def run_glm(Y, X, noise_model='ar1', bins=100, n_jobs=1, verbose=0):
 
     results : dict,
         Keys correspond to the different labels values
-        values are RegressionResults instances corresponding to the voxels.
+        values are SimpleRegressionResults instances corresponding to the voxels.
```
Comment (jeromedockes) [inline nistats/first_level_model.py]: this was correct, `run_glm` returns `RegressionResults`

File: nistats/first_level_model.py [hunk 2]
```diff
@@ -264,12 +264,12 @@ class FirstLevelModel(BaseEstimator, TransformerMixin, CacheMixin):
...
     ----------
-    labels : array of shape (n_voxels,),
+    labels_ : array of shape (n_voxels,),
         a map of values on voxels used to identify the corresponding model
 
-    results : dict,
+    results_ : dict,
         with keys corresponding to the different labels values
-        values are RegressionResults instances corresponding to the voxels
+        values are SimpleRegressionResults instances corresponding to the voxels
```
Comment (jeromedockes) [inline nistats/first_level_model.py]: `SimpleRegressionResults` if `minimize_memory` othewise `RegressionResults`

File: nistats/tests/test_dmtx.py [hunk 3]
```diff
@@ -466,7 +466,7 @@ def test_csv_io():
     path = 'design_matrix.csv'
     with InTemporaryDirectory():
         DM.to_csv(path)
-        DM2 = pd.DataFrame().from_csv(path)
+        DM2 = pd.DataFrame().read_csv(path)
```
Comment (jeromedockes) [inline nistats/tests/test_dmtx.py]: `pd.read_csv`

File: nistats/tests/test_dmtx.py [hunk 4]
```diff
@@ -466,7 +466,7 @@ def test_csv_io():
     path = 'design_matrix.csv'
     with InTemporaryDirectory():
         DM.to_csv(path)
-        DM2 = pd.DataFrame().from_csv(path)
+        DM2 = pd.read_csv(path)
```
Comment (jeromedockes) [inline nistats/tests/test_dmtx.py]:
```suggestion
        DM2 = pd.read_csv(path, index_col=0)
```

**Comment by kchawla-pi** [comment]
@jeromedockes Whenever you have the opportunity, lemme know if this is complete.

**Comment by jeromedockes** [comment]
> @jeromedockes Whenever you have the opportunity, lemme know if this is complete.

yes @kchawla-pi I think this is ready for merge

**Pull request merged**